package.json

```diff
@@ -47,6 +47,14 @@
                 "category": "Bazel",
                 "command": "bazel.clean",
                 "title": "Clean"
+            },
+            {
+                "command": "bazel.refreshBazelBuildTargets",
+                "title": "Refresh Bazel Build Targets",
```

Comment:
Should this use "category": "Bazel" and the title change accordingly like all the others?

Reply:
Yep, that was a merge resolution error. Thanks!